test/sql/test_array_fn/R/test_array_fn

```diff
@@ -4073,17 +4073,11 @@ None	None
 -- !result
 select d_6, d_5, all_match(d_6,d_5, (x,y)->x >y) from array_test order by pk;
 -- result:
-[4.000000,2.100000,100.000000,1.000000,4.300000,3.200000,10.000000,2.000000,3.000000,1.100000,-1.000000,-10.000000]	[4.000,2.100,3.000,1.100,4.300,3.200,-10.000,100.000,1.000,10.000,-1.000,2.000]	None
-[40.000000,20.100000,1000.000000,10.000000,40.300000,30.200000,100.000000,20.000000,3.000000,10.100000,-10.000000,-10.000000]	[40.000,20.100,30.000,10.100,40.300,30.200,-100.000,1000.000,1.000,100.000,-10.000,2.000]	0
-None	None	None
-[4.000000,2.100000,100.000000,null,1.000000,4.300000,3.200000,10.000000,2.000000,3.000000,1.100000,-1.000000,-10.000000]	[4.000,2.100,3.000,1.100,4.300,3.200,-10.000,100.000,1.000,10.000,-1.000,2.000]	None
+E: (1064, "Input array element's size is not equal in array_map().")
```

Comment:
Is it as expected behaviour?

Reply:
> Is it as expected behavior?

yes, because the two array‘s size is not equal, left is 12, right is 13